src/services/person/person.ts

```diff
@@ -0,0 +1,35 @@
+import { $auth } from '@mtfh/common';
+import { GetPersonRequestData, Person } from './person.types';
+import { config } from '../config';
+import { ResponseException } from '../../utils';
+
+const headers = {
+    'Content-Type': 'application/json',
+    'x-api-key': config.personApiKey,
+    'Authorization': `Bearer ${$auth.getValue().token}`,
+};
+
+export const getPersonById = async ({
```

Comment:
use useAxiosSWR like we have done in person mfe

Reply:
updated to use `useAxiosSWR`